_drafts/SC1-2-3+SC1-2-5-Video-description-track.md

```diff
@@ -0,0 +1,106 @@
+---
+name: Video element description track
+group: SC1-2-video-element
+description: |
+  This rule checks that non-streaming video elements with a description track have that track be descriptive.
```

Comment:
Sentence is not clear.

Reply:
Thank you. We will change the sentence. Probably to something along the lines of "This rule checks that description tracks that come with non-streaming video elements are descriptive."